crates/bevy_ecs/src/schedule/executor/multi_threaded.rs

```diff
@@ -147,13 +153,24 @@ impl SystemExecutor for MultiThreadedExecutor {
         self.unapplied_systems = FixedBitSet::with_capacity(sys_count);
 
         self.system_task_metadata = Vec::with_capacity(sys_count);
+        #[cfg(feature = "trace")]
+        {
+            self.system_task_spans = Vec::with_capacity(sys_count);
+            self.system_spans = Vec::with_capacity(sys_count);
+        }
         for index in 0..sys_count {
             self.system_task_metadata.push(SystemTaskMetadata {
                 archetype_component_access: default(),
                 dependents: schedule.system_dependents[index].clone(),
                 is_send: schedule.systems[index].is_send(),
                 is_exclusive: schedule.systems[index].is_exclusive(),
             });
+
+            #[cfg(feature = "trace")]
+            {
+                self.system_task_spans.push(info_span!("system_task", name = &*schedule.systems[index].name()));
```

Comment:
Wouldn't this break stack traces if or when they're enabled again?

Reply:
I think it should be ok as things are still properly parented in Tracy, but not 100% sure I understand which stack traces you're referring to.

Reply:
Tracy used to collect full stack traces as debug info, though it was disabled by default for performance reasons. That said it wouldn't be hard for a user to re-enable it in the Tracy client.